types/index.d.ts

```diff
@@ -0,0 +1,28 @@
+import Vue = require('vue')
+import { WatchOptions } from 'vue'
+import { Observable } from 'rxjs/Observable'
+
+export type Observables = Record<string, Observable<any>>
+declare module 'vue/types/options' {
+  interface ComponentOptions<V extends Vue> {
+    subscriptions?: Observables | ((this: V) => Observables)
+    domStreams?: string[]
+  }
+}
+
+declare module "vue/types/vue" {
+  interface Vue {
+    $observables: Observables;
+    $watchAsObservable(exprOrFn: string | Function, options?: WatchOptions): Observable<any>
+    $eventToObservable(event: string): Observable<any>
+    $subscribeTo<T>(
+      observable: Observable<T>,
+      next: (t: T) => void,
+      error: (e: any) => void,
+      complete: () => void): void
```

Comment:
Shouldn't `error` and `complete` be optional?

https://github.com/vuejs/vue-rx#subscribetoobservable-next-error-complete